src/main/java/com/github/adamyork/wiremock/transformer/VelocityResponseTransformer.java

```diff
@@ -57,6 +61,16 @@ public ResponseDefinition transform(final Request request,
                       context.put("requestAbsoluteUrl", request.getAbsoluteUrl());
                       context.put("requestUrl", request.getUrl());
                       context.put("requestMethod", request.getMethod());
+					  context.put("numberTool", new NumberTool());
```

Comment:
according to the documentation for velocity tools this shouldn't be necessary, as the ToolManager comes defaults configured with the Generic Tools , which include the NumberTool. If there is an issue resolving methods of the object via the templates , that will need to be looked into separately.

Reply:
Hmm.  That's strange.  I couldn't get to resolve it.  Yes, it can be deferred to separate issue.